.github/workflows/master.yml

```diff
@@ -87,14 +87,15 @@ jobs:
       - name: Import check
         run: |
           source .venv/bin/activate
+          pip install 'pyspark>=3.5'
```

Comment:
I believe this is the problem. We don't need to install pyspark here :) Only in the optuna test suite.